Xamarin.Forms.Platform.iOS/Renderers/ImageRenderer.cs

```diff
@@ -120,6 +120,27 @@ public interface IImageSourceHandler : IRegisterable
 		Task<UIImage> LoadImageAsync(ImageSource imagesource, CancellationToken cancelationToken = default(CancellationToken), float scale = 1);
 	}
 
+	internal static class ImageSourceExtensions
+	{
+		public static async Task<UIImage> GetNativeImageAsync(this ImageSource source)
```

Comment:
I extracted the simple logic of taking an ImageSource and getting a UIImage - this chunk is duplicated in at least 20 places. Hopefully, moving forward, we can use this instead. This is mostly for image buttons, icons, etc. The `<Image />` element has more functionality for loading and progress tracking.